src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,53 @@
+package lotto.domain;
+
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber> {
+
+	private static final int MIN_NUMBER = 1;
+	private static final int MAX_NUMBER = 45;
+
+	private final int lottoNumber;
+
+	public LottoNumber(int number) {
+		if (isNotLottoNumber(number)) {
+			throw new IllegalArgumentException();
+		}
```

Comment:
아래처럼 검증을 분리해보면 좋겠네요.

```suggestion
	public LottoNumber(int number) {
		validateLottoNumber(int number);
```